site/src/components/gradient-card.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Gradient card
```

Comment:
Is there a Figma with a design available for this component? If so, it needs to be attached to this page by adding `figmaEmbed: {url}` meta to this section.

If not, please ask the designer you're working with on this to create it in Atlas's Figma library - https://www.figma.com/file/uVA2amRR71yJZ0GS6RI6zL/%F0%9F%8C%9E-Atlas-Design-Library?type=design&node-id=2911-12063&t=zuIxFGBM7ItdC1Ji-0

Reply:
Yes for sure, it can be added later. Just important to keep Figma library aligned with what we have here :)

Feel free to resolve after tasks are created.

Reply:
Created! https://ceapex.visualstudio.com/Engineering/_workitems/edit/831925